deps.windows/70-vpl.ps1

```diff
@@ -0,0 +1,87 @@
+param(
+    [string] $Name = 'vpl',
+    [string] $Version = 'v2023.3.0',
+    [string] $Uri = 'https://github.com/oneapi-src/oneVPL.git',
+    [string] $Hash = 'e12ace9761bb52786409e830f619916b86e87fc5'
```

Comment:
Add `[switch] $ForceStatic = $true` here.

Reply:
I've added this via force push.